src/screens/GroupInfo/hooks/useGroupInfo.js

```diff
@@ -570,7 +573,8 @@ const useGroupInfo = (channelId = null) => {
     setIsAnonymousModalOpen,
     blockModalRef,
     isFetchingAllowAnonDM,
-    isLoadingInitChat
+    isLoadingInitChat,
+    isLoadingAddMember
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The inclusion of `isLoadingAddMember` in the returned object is necessary for other components to access and monitor the loading state when adding members.
```diff
+ isLoadingAddMember
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->